stripe-core/src/main/java/com/stripe/android/core/exception/InvalidResponseException.kt

```diff
@@ -0,0 +1,22 @@
+package com.stripe.android.core.exception
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.core.StripeError
+
+/**
+ * A [StripeException] indicating that invalid parameters were used in a response.
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class InvalidResponseException(
```

Comment:
Since this exception is meant to be used across API requests in different SDKs, is it possible to better specify what an invalid parameter in a response is and when to throw it? Is it a response parsing error?

Reply:
Sure, added some more descriptions to the doc. Was planning to use this when an enum is not ready and when some non-null value is null